### DISCUSSION TITLE
Form.Item下面包一个Table，Table的单元格里面渲染有表单，这竟然是有效的，Form拿到的值是一个数组，数组的每一项是表格的每一行的数据

### DISCUSSION THREAD

**Discussion by ChenKun1997** [ant-design/ant-design#43524] [Q&A]
Form.Item下面包一个Table，Table的单元格里面渲染有表单，这竟然是有效的，Form拿到的值是一个数组，数组的每一项是表格的每一行的数据，现在我想做表单校验，校验只能写在Form.Item上，没办法针对表格中的具体每个表单做校验，有没有解决方案

**Comment by zombieJ** [comment]
听起来你需要的是 ProEditTable:
https://procomponents.ant.design/components/editable-table
